pkg/security/probe/probe_windows.go

```diff
@@ -1386,6 +1386,11 @@ func (p *Probe) Origin() string {
 	return ""
 }
 
+// EnableEnforcement sets the enforcement mode
+func (p *WindowsProbe) EnableEnforcement(state bool) {
+	p.processKiller.enabled = state
```

Comment:
shouldn't this use the function `SetState`, i.e. use the lock, otherwise I'm worried about races